.github/workflows/build.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - uses: abatilo/actions-poetry@v2.1.4
         with:
-          poetry-version: 1.2.2
+          poetry-version: 1.4.0
```

Comment:
Can you use the following action https://github.com/exasol/integration-test-docker-environment/blob/main/.github/actions/prepare_poetry_env/action.yml and copy it to this repo

Reply:
and use it also it this file 
https://github.com/exasol/transformers-extension/blob/62e126ed475123433eb6a23ca2f42857002625fc/.github/workflows/release_droid_upload_github_release_assets.yml

Reply:
done